### PR TITLE
facs: add all conditions button, two conditions per line

### PR DESCRIPTION
**Pull request by annagav**
#314, #313

### CODE REVIEW

File: html_app/ui/FacsView.js [hunk 1]
```diff
@@ -141,6 +141,43 @@ scb.ui.static.FacsView.scb_f_facs_select_lysate_type = function (element, event)
...
+                parsed.facs.lanes_list.start({
+                    kind: kind,
+                    conditions: condition,
+                    cell_treatment_id: cell_treatment_id,
+                    experiment_id: parsed.experiment.id
+                });
+            }
+        });
+        /* want to remove any lanes that did not have condition selected */
+        var lanes_no_cond = _.filter(lanes, function(lane){return lane.conditions == null});
```
Comment (carsongee) [inline html_app/ui/FacsView.js]: nit: maybe put return on new line.

File: html_app/ui/FacsView.js [hunk 2]
```diff
@@ -141,6 +141,43 @@ scb.ui.static.FacsView.scb_f_facs_select_lysate_type = function (element, event)
...
+    var lanes = _.filter(parsed.facs.lanes_list.list, function(lane) {
+        return cell_treatment_id == lane.cell_treatment_id
+    });
+    var facs_kinds = parsed.assignment.template.facs_kinds;
+    _.each(_.keys(facs_kinds), function(kind){
+        var conditions = _.keys(facs_kinds[kind].conditions);
+
+        _.each(conditions, function(condition){
+            /* find if a lane exists with this condition */
+            var lane = _.find(lanes, function(lane){ return lane.conditions === condition});
```
Comment (carsongee) [inline html_app/ui/FacsView.js]: same nit.

File: html_app/ui/facs.soy [hunk 3]
```diff
@@ -301,31 +305,37 @@ FACS Main View
...
-            {if $r.is_sample_enabled}			 
-            	<input type="radio"  checked>
-            	    <span class='scb_s_facs_cell_treatment_text'>Fixed</span>
-            	</input>
-            {else}
-                <input type="radio" disabled>
-                    <span class='scb_s_facs_cell_treatment_text scb_s_microscopy_dim_conditions'>Fixed</span>
-                </input>
-            {/if}
+            <td class='scb_s_facs_samples_table_td {if $r.display_sample}{else}scb_s_experiment_setup_table_border {/if}
```
Comment (carsongee) [inline html_app/ui/facs.soy]: It seems weird to control css classes with inline if statements, but it is clear enough to understand.  I almost wonder if something like:

``` html
<script type="text/javascript">
{if not $r.display_sample}
$(.....).addClass('scb_s_experiment_setup_table_border')
{/if}
</script>
```

to do it one place would save a lot of code.  Also, is there not a `{if not ...` in this template or similar?  The `else` seems unnecessary for all of these.

**Comment by carsongee** [comment]
I could also use some help verifying this addresses the issues in addition to the comments.

Thanks

**Comment by pdpinch** [comment]
I can confirm this is working as expected, and addresses the issues (in flow cytometry)

The issue mentions a similar UI for microscopy, but this PR doesn't address that.

**Comment by pdpinch** [comment]
This looks ready to merge to me.

**Pull request merged**